tests/admin_changelist/test_date_hierarchy.py

```diff
@@ -22,6 +22,9 @@ def assertDateParams(self, query, expected_from_date, expected_to_date):
         request.user = self.superuser
         changelist = EventAdmin(Event, custom_site).get_changelist_instance(request)
         _, _, lookup_params, _ = changelist.get_filters(request)
+        sql = (str(changelist.queryset.query))
+        self.assertIn('>= %s' % expected_from_date.date().isoformat(), sql)
+        self.assertIn('< %s' % expected_to_date.date().isoformat(), sql)
```

Comment:
I don't think testing against the invalid (missing quotes) introspection SQL is necessary given the assertions below already fail without the `get_filters` changes.

Reply:
I removed these assertions.